cypress/e2e/1-auth/sign-up.cy.ts

```diff
@@ -36,3 +36,11 @@ describe("Sign Up", () => {
     cy.get('[data-testid="email-error"]').should("be.visible");
   });
 });
+
+describe.only("check redirects", () => {
```

Comment:
Please remove `only` because it skips all previous tests. 

Also, please start the name in `describe` and `it` with capital letters 😊